modules/creating-a-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="creating-a-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+As a cluster administrator, you can create a custom Ingress Controller to manage DNS record lifecycle.
```

Comment:
```suggestion
As a cluster administrator, you can create a custom Ingress Controller with `Unmanaged` DNS management policy.
```

Reply:
I think it is better for this section to explicitly target "Creating a custom ingress controller with `Unmanaged` DNS management policy".